model/service.go

```diff
@@ -40,15 +40,14 @@ type ServiceDiscovery interface {
 // Service describes an Istio service
 type Service struct {
```

Comment:
Note to self. This file needs to be re-organized.. Structs first, interfaces later, for better code readability. And needs heavy commenting, mapping out the structs to platform features in kube and mesos.